helpers/helpers.go

```diff
@@ -62,6 +63,8 @@ func KeyLength(key interface{}) int {
 		return ecdsaKey.Curve.Params().BitSize
 	} else if rsaKey, ok := key.(*rsa.PublicKey); ok {
 		return rsaKey.N.BitLen()
+	} else if _, ok := key.(ed25519.PublicKey); ok {
+		return ed25519.PublicKeySize
```

Comment:
Maybe here there is a way to use the size of the instantiation of the type rather than using the constant..

Reply:
ed25519 keys are always the same length, so the constant is ok